src/core/reference.js

```diff
@@ -20,6 +28,14 @@
  */
 
 /**
+ * Creates and names a new constant variable. A variable is a container for a value.
```

Comment:
The phrase "constant variable" is potentially confusing. I stumble with this too, but maybe something like "Creates and names a new constant. Like a variable, a constant is a container for a value, however constants cannot be changed once they are declared.